components/script/fetch.rs

```diff
@@ -261,6 +261,7 @@ impl FetchResponseListener for FetchContext {
 
     fn process_response_chunk(&mut self, mut chunk: Vec<u8>) {
         self.body.append(&mut chunk);
+        self.stream_chunk(chunk.as_slice());
```

Comment:
Since this is a method on FetchContext and not Response, you need to use `aelf.respinse_object.root()` to obtain a `DomRoom<Response>` object that you can use to call Response methods.

Reply:
Done